sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -79,22 +79,23 @@ The following examples show common scenarios using the `client` [created above](
 To analyze a conversation, we can then call the `client.AnalyzeConversation()` method which takes the project name, deployment name, and query as parameters.
 
 ```C# Snippet:ConversationAnalysis_AnalyzeConversation
+ConversationsProject conversationsProject = new ConversationsProject("Menu", "production");
+
 Response<AnalyzeConversationResult> response = client.AnalyzeConversation(
-    "Menu",
-    "production",
-    "We'll have 2 plates of seared salmon nigiri.");
+    "We'll have 2 plates of seared salmon nigiri.",
+    conversationsProject);
 
 Console.WriteLine($"Top intent: {response.Value.Prediction.TopIntent}");
 ```
 
 The specified parameters can also be used to initialize a `ConversationAnalysisOptions` instance. You can then call `AnalyzeConversation()` using the options object as a parameter as shown below.
 
 ```C# Snippet:ConversationAnalysis_AnalyzeConversationWithOptions
+ConversationsProject conversationsProject = new ConversationsProject("Menu", "production");
 AnalyzeConversationOptions options = new AnalyzeConversationOptions(
-    "Menu",
-    "production",
     "We'll have 2 plates of seared salmon nigiri.");
-Response<AnalyzeConversationResult> response = client.AnalyzeConversation(options);
+
+Response<AnalyzeConversationResult> response = client.AnalyzeConversation(conversationsProject, options);
```

Comment:
To be consistent with Question Answering, `query` needs to be moved out and probably needs to be renamed as well. `AnalyzeConversationOptions` becomes completely optional. As you can see in QA, I still pack the `question` in it internally because that's what the generated REST client requires, but it should not be exposed publicly otherwise.

Reply:
Ohh thanks for pointing that out. I missed the fact that [this overload](https://github.com/Azure/azure-sdk-for-net/blob/c429da7ddcd3ecf1c6f500d24660c98620a5bc79/sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/QuestionAnsweringClient.cs#L105) is private.